src/models/applicationModel.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
```

Comment:
please add @hapi/joi  validation for user inputs. it should return the validation function